pipeline/metadata/test_flatten_base.py

```diff
@@ -165,3 +172,31 @@ def test_parse_received_data_https(self) -> None:
     blockpage_matcher = BlockpageMatcher()
     parsed = flatten_base.parse_received_data(blockpage_matcher, received, True)
     self.assertEqual(parsed, expected)
+
+  def test_parse_cert_invalid(self) -> None:
```

Comment:
You can test that the correct warning log is being produced [like this](https://github.com/censoredplanet/censoredplanet-analysis/blob/ba07ca757c241cbbf9895c8d958a5cea1dc881e7/pipeline/metadata/test_flatten.py#L16).

Reply:
done